README.md

```diff
@@ -113,6 +113,17 @@ puppetserver::config::bootstrap { 'puppetlabs.services.ca.certificate-authority-
 }
 ```
 
+A Puppetserver >= 2.5.0 ca.cfg entry.
```

Comment:
This is a bit confusing as the define actually does this for us, so it is not needed here.

Reply:
the define is not aware of the used puppetserver version, so it will not be able to choose the correct path. You need to manually set this.

If $puppetserver::version would always contain a version number we could use that, but it is needed to also support the default of 'present' that can be any version.

Reply:
Technical it would be possible to use `generate()` to get the used puppetserver version.
Usually I would prefer to avoid generators for the unneeded additional load on the puppet masters.

Reply:
Agreed on using the generator function.